### PR TITLE
First step in creating zero-to-one

### PR DESCRIPTION
**Pull request by thomascothran**
A minimal example of how to set up and run zero.

Demo how to:

1. Get a build set up
2. Run browser tests
3. Use css stylesheets
4. With hot reload

### CODE REVIEW

**Comment by thomascothran** [comment]
There's a few things I'd like to change before moving this out of draft status:

1. I'd like to SSR the component. This requires setting up a backend server. I'll probably stick with ring and jetty, no router.
2. Clean up tests a bit more. 
3. CI tests. Karma?

File: examples/zero-to-one/test/zero_to_one/client_test.cljs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   :after #(async done
+                  (.remove (get-incrementing-button))
+                  (reset! client/clicks* 0)
+                  (done))})
+
+(deftest incrementing-button-is-registered
+  (is (js/customElements.get "incrementing-button")))
+
+(deftest incrementing-button-test
+  (async done
```
Comment (raystubbs) [inline examples/zero-to-one/test/zero_to_one/client_test.cljs:41]: Neat, didn't know about this 😅

**Review by raystubbs — APPROVED** [review]
🔥 🚀

**Comment by raystubbs** [comment]
Gonna close this without merging since it's basis is before a lot of major breaking changes that have occurred since.  But I would like to migrate it at some point.

**Pull request closed without merging**